rasa/core/validator.py

```diff
@@ -187,13 +188,42 @@ def verify_utterances_in_stories(self, ignore_warnings: bool = True) -> bool:
 
         return everything_is_alright
 
+    def verify_story_structure(
+        self, ignore_warnings: bool = True, max_history: int = 5
+    ) -> bool:
+        """Verifies that bot behaviour in stories is deterministic."""
+
+        logger.info("Story structure validation...")
+        logger.info(f"Assuming max_history = {max_history}")
+
+        trackers = TrainingDataGenerator(
+            self.story_graph,
+            domain=self.domain,
+            remove_duplicates=False,  # ToDo: Q&A: Why not remove_duplicates=True?
+            augmentation_factor=0,
+        ).generate()
+
+        # Create a list of `StoryConflict` objects
+        conflicts = StoryConflict.find_conflicts(trackers, self.domain, max_history)
+
+        if len(conflicts) == 0:
+            logger.info("No story structure conflicts found")
+        else:
+            for conflict in conflicts:
+                logger.warning(conflict)
+
+                # For code stub to fix the conflict in the command line,
```

Comment:
what's this?

Reply:
I had started to implement an interactive auto-correct feature. I want to have a note here to know where that code was, in case I want to continue that.

Reply:
@wochinge is that ok?

Reply:
Do you really need that? 😄 How about creating an issue for the interactive auto-correct feature?

Reply:
Ok. Done here: https://github.com/RasaHQ/rasa/issues/5082